MAUI/Cartesian-Charts/StackedArea100.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: StackedArea100 Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about StackedArea100 chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# StackedArea100 Chart in .NET MAUI Chart
+
+The stacked area 100% chart enables users to visually represent data points as stacked areas. In this chart, the areas are stacked on top of each other to indicate the cumulative value of the data points.
```

Comment:
@snehakPV 
Both stacked area and 100 stacked area description are similar, modify like for stacked area 100 - cumulative value of the data points to 100%.